posthog/api/test/test_personal_api_keys.py

```diff
@@ -440,6 +442,23 @@ def test_works_with_routes_missing_action(self):
         )
         assert response.status_code == status.HTTP_200_OK
 
+    def test_works_with_session_recording_snapshots(self):
```

Comment:
I don't think this test should be here. This is about generally testing the API key functionality. If you just want to add a scope and test that it works then it should be part of the api tests for that endpoint (or just not bother as we know generally that adding the `required_scopes` property works.